core/src/main/java/com/klaytn/caver/rpc/RPC.java

```diff
@@ -40,14 +40,20 @@
      */
     public Net net;
 
+    /**
+     * The API class related governance.
+     */
+    public Governance gov;
```

Comment:
[caver-js](https://github.com/klaytn/caver-js/blob/9175a4f24821d6693070c9b73ad07177aa16100c/packages/caver-rpc/src/index.js#L52) use the field name `governance`, not `gov`.

Considering usability, it will be better to match the variable name.